testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueryFramework.java

```diff
@@ -104,7 +104,7 @@ protected abstract QueryRunner createQueryRunner()
     public final void close()
             throws Exception
     {
-        try (afterClassCloser) {
+        try (AutoCloseableCloser ignored = afterClassCloser) {
```

Comment:
google/error-prone#3321

Reply:
Can you add a code-level comment with a link?

Reply:
This has been fixed in Error Prone 1.15.0, so this change is gone from the PR.